### PR TITLE
Add basic validation to crds

### PR DESCRIPTION
**Pull request by polskikiel**
This PR is a 
 - [x] Feature Implementation
 - [ ] Bug Fix
 - [ ] Documentation

**What I couldn't validate in those crds:**
- ServiceInstances object references validation `pkg/apis/servicecatalog/validation/instance.go#L90`
- Brokers `catalog restrictions` validation

Please leave this checklist in the PR comment so that maintainers can ensure a good PR.

Merge Checklist:
   - [x] Chart changes

- https://github.com/kyma-project/kyma/issues/2798

### CODE REVIEW

File: charts/catalog/templates/crds.yaml [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+    singular: clusterserviceplan
+    kind: ClusterServicePlan
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
+          - "clusterServiceClassRef"
```
Comment (mszostok) [inline charts/catalog/templates/crds.yaml:333]: why it's required?

File: charts/catalog/templates/crds.yaml [hunk 2]
```diff
@@ -0,0 +1,608 @@
...
+          - "clusterServiceBrokerName"
```
Comment (mszostok) [inline charts/catalog/templates/crds.yaml:334]: why it's required?

File: charts/catalog/templates/crds.yaml (hunk 1 above)
Comment (polskikiel) [reply charts/catalog/templates/crds.yaml:333]: https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/apis/servicecatalog/validation/serviceplan.go#L110

**Review by mszostok — APPROVED** [review]

File: charts/catalog/templates/crds.yaml (hunk 1 above)
Comment (mszostok) [reply charts/catalog/templates/crds.yaml:333]: ok, so let's merge it like this and in https://github.com/kyma-project/kyma/issues/2792 we will recheck what kind of validation choose based on user experience :)

**Pull request merged**